OrbitGl/LiveFunctionsDataView.cpp

```diff
@@ -149,6 +160,14 @@ std::vector<std::string> LiveFunctionsDataView::GetContextMenu(
   std::vector<std::string> menu;
   if (enable_select) menu.emplace_back(MENU_ACTION_SELECT);
   if (enable_unselect) menu.emplace_back(MENU_ACTION_UNSELECT);
+
+  // For now, these actions only make sense when one function is selected, 
+  //     so we don't show them otherwise.
```

Comment:
Remove indentation in comment. Should be 

// For now, these actions only make sense when one function is selected, 
// so we don't show them otherwise.